sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -237,11 +239,13 @@ export async function saveFakeApplication(
     institutionLocation?: InstitutionLocation;
     student?: Student;
     program?: EducationProgram;
+    offering?: EducationProgramOffering;
   },
   options?: {
     applicationStatus?: ApplicationStatus;
     offeringIntensity?: OfferingIntensity;
     applicationData?: ApplicationData;
+    applicationProgramYearId?: number;
```

Comment:
Please adjust it accordingly: https://github.com/bcgov/SIMS/pull/3276/files#r1600534417